python/cudf/cudf/core/series.py

```diff
@@ -3773,9 +3853,7 @@ def quarter(self):
 
     @_cudf_nvtx_annotate
     def isocalendar(self):
-        """
-        Returns a DataFrame with the year, week, and day
-        calculated according to the ISO 8601 standard.
+        """Return a DataFrame with the year, week, and day calculated according to the ISO 8601 standard.
```

Comment:
I'd wrap this (so it doesn't break E501) and instead choose to break whichever rule requires the brief to be one line, if that's equally viable.